libcloudforensics/providers/aws/forensics.py

```diff
@@ -13,9 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Forensics on AWS."""
-from typing import TYPE_CHECKING, Tuple, List, Optional, Dict, Any
+from typing import TYPE_CHECKING, Tuple, List, Optional, Dict
 
-from libcloudforensics.providers.aws.internal.common import UBUNTU_1804_AMI, LOGGER  # pylint: disable=line-too-long
+from libcloudforensics.providers.aws.internal.common import UBUNTU_1804_FILTER, LOGGER  # pylint: disable=line-too-long
```

Comment:
I suggest that for imports exceeding line length, we do

```
# pylint: disable=line-too-long
import ...
# pylint: enable=line-too-long
```

Instead of having one for each line. WDYT @giovannt0 @sa3eed3ed ?

Reply:
I don't mind either (unless there are several `import` statements which need the annotation, in which case I would be in favour of the above solution)

Reply:
gonna leave it as it is for now as it's 1 line :).